From: Fengguang Wu <fengguang.wu@intel.com>
Date: Wed, 7 Mar 2018 02:23:23 +0800
Subject: enic: fix boolreturn.cocci warnings
Patch-mainline: v4.17-rc1
Git-commit: 7d99429afa9393968261a425b1e1b57298e48a75
References: bsc#1113431

drivers/net/ethernet/cisco/enic/vnic_dev.c:1294:9-10: WARNING: return of 0/1 in function 'vnic_dev_capable_udp_rss' with return type bool

 Return statements in functions returning bool should use
 true/false instead of 1/0.
Generated by: scripts/coccinelle/misc/boolreturn.cocci

Fixes: 48398b6e7065 ("enic: set UDP rss flag")
CC: Govindarajulu Varadarajan <gvaradar@cisco.com>
Signed-off-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/cisco/enic/vnic_dev.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/cisco/enic/vnic_dev.c
+++ b/drivers/net/ethernet/cisco/enic/vnic_dev.c
@@ -1291,7 +1291,7 @@ bool vnic_dev_capable_udp_rss(struct vni
 
 	err = vnic_dev_cmd(vdev, CMD_CAPABILITY, &a0, &a1, wait);
 	if (err || !a0)
-		return 0;
+		return false;
 
 	rss_hash_type = (a1 >> NIC_CFG_RSS_HASH_TYPE_SHIFT) &
 			NIC_CFG_RSS_HASH_TYPE_MASK_FIELD;
